### ISSUE TITLE
Fix debian packages

### ISSUE THREAD

**Issue by elsirion** [fedimint/fedimint#3766]
Our debian packages are broken in a few ways:

1. The client package is empty (can likely be removed, `fedimint-cli` is included in the `fedimint` package)
2. The `fedimint` package installs the necessary binaries but doesn't correctly link them to `/bin`

```
root@04a40849f8dc:~# dpkg -c fedimint-pkgs-bin-fedimint-pkgs_1.0_amd64.deb 
drwxr-xr-x 0/0               0 1980-01-01 00:00 ./
drwxrwxrwx 0/0               0 1980-01-01 00:00 ./bin/
lrwxrwxrwx 0/0               0 1980-01-01 00:00 ./bin/fedimint-pkgs -> /nix/store/0qkacykmwmd7vvx4gzsx6kdqn7zfqp02-fedimint-pkgs/bin/fedimint-pkgs
drwxrwxrwx 0/0               0 1980-01-01 00:00 ./nix/
drwxrwxrwx 0/0               0 1980-01-01 00:00 ./nix/store/
…

root@04a40849f8dc:~# ls /nix/store/0qkacykmwmd7vvx4gzsx6kdqn7zfqp02-fedimint-pkgs/bin/
fedimint-cli  fedimint-dbtool  fedimintd
```

Instead of the link `./bin/fedimint-pkgs -> /nix/store/0qkacykmwmd7vvx4gzsx6kdqn7zfqp02-fedimint-pkgs/bin/fedimint-pkgs` there should be three links to the actual binaries contained in `/nix/store/0qkacykmwmd7vvx4gzsx6kdqn7zfqp02-fedimint-pkgs/bin`.

**Comment by dpc** [comment]
```
> nix bundle .#fedimintd -o fedimintd.bundled
> ./fedimintd.bundled --help
Usage: fedimintd [OPTIONS] --data-dir <DATA_DIR>
...
```

That much works, possibly. Let's see dpkg.

**Comment by elsirion** [comment]
Yes, the normal binaries worked for me too (even though the first start is comically slow).

**Comment by dpc** [comment]
The tool seems to assume that the name of the output is the name of the binary. I have a fix coming that just does it this way, and hopefully good enough for now.

**Comment by dpc** [comment]
Fixed in #3836
